tests/login/ready.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+rlJournalStart
+    rlPhaseStartSetup
+        rlRun "pushd data"
+    rlPhaseEnd
+
+    rlPhaseStartTest "Positive login test for local"
+        rlRun -s "tmt run -a provision -h local login -c exit" 0 \
+            "Should be allowed to login into guest which is local"
+    rlPhaseEnd
+
+    rlPhaseStartTest "Positive login test for container"
+        rlRun -s "tmt run -a provision -h container login -c exit" 0 \
+            "Should be allowed to login into guest which is a container"
+    rlPhaseEnd
+
+    rlPhaseStartTest "Positive login test for artemis"
+        rlRun -s "tmt run -a provision -h artemis login -c exit" 0 \
+            "Should be allowed to login into guest which is artemis"
+    rlPhaseEnd
+
+    rlPhaseStartTest "Negative login test 01 for virtual"
+        rlRun -s "tmt run -a provision -h virtual -i FOOOOOOOOOOOOOO login -c exit" 2 \
```

Comment:
There is no change here - broken tmt exists with '2' as well as fixed one.
Please check for output as well. I'd say "Starting interactive shell" should not be present (`rlAssertNotGrep "Starting interactive shell" "$rlRun_LOG"`), neither "Failed to push workdir" and add some grep for text which should be there - I'd assume in this case "Could not map .* to compose"

Reply:
Updated.